src/Accordion/accordion.js

```diff
@@ -82,9 +82,9 @@ class Accordion extends Component {
     }
 
     render() {
-        const { className } = this.props;
+        const { className, accordion } = this.props;
         return (
-            <div className={className}>
+            <div role={accordion ? 'tablist' : null} className={className}>
```

Comment:
From what I understand, `accordion` defines "Open only one item at a time or not". Shouldn't there be another role defined as well when `accordion` is false?

Reply:
I don't think so. They are just expandable buttons otherwise. And this is just a container.
I will double check though.